README.md

```diff
@@ -1,21 +1,21 @@
-# Hello World!
-
-My name is Agata and this is my Portfolio repository!
-
-This repository is a collection of my work and experience I have gathered in the last few months learning Python and how to code.
-
-## My projects
-
-=> [Adventure Game](https://github.com/Agata-Gizinska/Adventure_game/tree/master)
-
-=> [Class Register](https://github.com/Agata-Gizinska/Class_register/tree/master)
-
-=> [Dice Game](https://github.com/Agata-Gizinska/Dice_Game/tree/master)
-
-=> [File Sorter](https://github.com/Agata-Gizinska/File_Sorter/tree/1b348c7d6d6733490207c6cbd922c194b7ded702)
-
-=> [Web Scrapper for currency rates](https://github.com/Agata-Gizinska/Web_scrapper_NBP/tree/master)
-
-## Technologies and setup
-
-All requirements for a setup are listed in the readme file for each project.
+# Hello World!
+
+My name is Agata and this is my Portfolio repository!
+
+This repository is a collection of my work and experience I have gathered in the last few months learning Python and how to code.

```

Comment:
This would imply you have just a few months of hands-on experience in python. You have more, those are just some samples you feel like sharing. Also it would be untrue for any amount of months > 12

Try:
"This is my publishing space for any project I feel is somewhat complete and usable."

Reply:
ok